app/controllers/api/v1/items/find_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::Items::FindController < ApplicationController
+  def show
+    respond_with Item.find_by(item_params)
+  end
+
+  def index
+    respond_with Item.where(item_params).find_each do |item|
+      item
+    end
```

Comment:
we don't really need this find_each; we can just call where and get back an array of al the matching records